### PR TITLE
damage display works

### PR DESCRIPTION
**Pull request by BenRodProd**
if someone sees the issue, why two questions are triggered now, i would be thankful... but the userStory "damage display" works...

### CODE REVIEW

File: components/Answer.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ import { useState, useEffect, useRef } from "react";
 
 const InputFieldLayout = styled.div`
   display: flexbox;
```
Comment (botacode-gh) [inline components/Answer.js:5]: better change this to `display: flex` before CJ sees it

File: pages/library/[id].js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { useRouter } from "next/router";
 import styled from "styled-components";
-import { useState } from "react";
+import { useState, useEffect } from "react";
```
Comment (botacode-gh) [inline pages/library/[id].js:3]: You could also include `useRef` in this line

**Review by botacode-gh — APPROVED** [review]
Some tiny comments, otherwise LvGTM!

File: components/Answer.js (hunk 1 above)
Comment (BenRodProd) [reply components/Answer.js:5]: Ohhh... right!

**Pull request merged**